ui-src/src/app/configurations/configurations.component.ts

```diff
@@ -60,35 +60,41 @@ export class ConfigurationsComponent implements OnInit {
             break;
           }
           case 400: {
-            this.toastr.error("Some of the Fields do not Conform to the Schema!", "An Error Occurred!");
+            this.toastr.error(
+              "Some of the Fields do not Conform to the Schema!",
```

Comment:
"Some of the fields do not conform to the schema"

Reply:
@ColmCooney @angerooney11  @frankquinn Can we use sentence case on all messages as well. https://www.snappysentences.com/sentence-case-v-title-case/. @ColmCooney Just change this message to the correct sentence case. I will add a bug to handle the rest. So that you can move on.